src/Microsoft.ML.StandardLearners/Standard/SdcaRegression.cs

```diff
@@ -25,6 +25,12 @@
 namespace Microsoft.ML.Runtime.Learners
 {
     /// <include file='doc.xml' path='doc/members/member[@name="SDCA"]/*' />
+    /// <example>
+    /// <format type="text/markdown">
+    /// <![CDATA[
+    ///  [!code-csharp[SDCA Regression](../../../docs/samples/Microsoft.ML.Samples.StaticPipe/Trainers.cs?range=5-8,12-14,25-79) "The SDCA regression example."]
```

Comment:
We aren't actually using the class `SdcaRegressionTrainer` in this example code. I think that would be an anti-pattern in the docs. If I want to see an example of using class `Foo`, the example should have the class `Foo` in it, IMO.